lib/direct_select_item.dart

```diff
@@ -128,6 +128,7 @@ class DirectSelectItemState<T> extends State<DirectSelectItem<T>>
     } else {
       return Material(
         child: Container(
+          color: Theme.of(context).scaffoldBackgroundColor,
```

Comment:
Maybe it's better to make this color field configurable by passing it at the `DirectSelectItem` constructor? @iyatsouba